Source/WTF/wtf/cocoa/NSURLExtras.mm

```diff
@@ -115,7 +115,7 @@ static String decodePercentEscapes(const String& string)
 {
     auto trimmed = adoptNS([string mutableCopy]);
     CFStringTrimWhitespace((__bridge CFMutableStringRef)trimmed.get());
-    return trimmed;
+    return adoptNS([trimmed copy]);
```

Comment:
I don't think we want to do this. Why make a copy?

Reply:
Does not matter; NSMutableArray is a NSArray subclass; the caller will never know, this is a needless copy.